### ISSUE TITLE
Добавьте пожалуйста инструмент usdrub tom tod

### ISSUE THREAD

**Issue by verbadim** [SBProX/SBProX-Dev#1478]
Здравствуйте, не могли бы вы добавить инструмент usdrub tom,желательно и tod. Программа отличная,но очень не хватает объемов usdrub tom. Контракт SI(usd-rub) самый ликвидный контракт московской биржи на данный момент, контракт наиболее популярен у спекулянтов.Так, что будет многим полезно. Знаю,что вы работаете над добавлением акций РФ. Вполне можно начать добавление базовых активов с валюты, вы получите обратную связь от пользователей, фьючерс на доллар рубль торгуют очень многие.

**Comment by longist777** [comment]
Очень хорошее предложение. На ФОРТС не много инструментов с хорошими объёмами и сишка один из них.

**Comment by verbadim** [comment]
Да, в том то и дело. Си это единственный фьючерс с базовым активом и оборотом в 200 млрд в день, с большим количеством понятных ситуаций. Можно добавить тод и том и многие пользователи тайгера и атаса перейдут на сбпро. Добавить один, два актива это не 250 акций мосбиржи. Добавив том и тод обратной связи будет больше,чем если добавить все акции мосбиржи. А дальше уже, успешно внедрив два инструмента можно и остальные добавлять,если нужно.

**Comment by Petrov007** [comment]
Поддерживаю

**Comment by smart3353** [comment]
лайк

**Comment by IURII79** [comment]
Лайк

**Comment by kazan-trader** [comment]
поддерживаю
